Sources/OktaIdx/IDXRemediation.swift

```diff
@@ -90,6 +90,21 @@ extension IDXClient {
             super.init()
         }
         
+        public override var description: String {
+            let logger = DebugDescription(self)
+            let components = [
+                logger.address(),
+                "\(#keyPath(type)): \(type)",
+                "\(#keyPath(name)): \(name)"
```

Comment:
We shouldn't need to print `name` since that's just a string version of the enum `type`

Reply:
Done